salt_stack/salt/lambda-dev/files/makedomainenv

```diff
@@ -68,18 +75,33 @@ else
    fi
 
    cd ~/virtualenvs/${1}/local/lib/python3.6/dist-packages
-   chmod -R 077 *
+   chmod -R 0777 *
    zip -r9 ~/lambdazips/multilambda.${1}.zip *
 
    cd ~/virtualenvs/${1}/local/lib/python3.6/site-packages
-   chmod -R 077 *
+   chmod -R 0777 *
    zip -r9 ~/lambdazips/multilambda.${1}.zip *
 
    cd ~/virtualenvs/${1}/lib64/atlas
    zip -r9 ~/lambdazips/multilambda.${1}.zip *
 
    cd ~/virtualenvs/${1}/local/lib/python3.6/site-packages/lambda
    zip -g ~/lambdazips/multilambda.${1}.zip lambda_loader.py
+   zip -g ~/lambdazips/multilambda.${1}.zip write_s3_index_lambda.py
```

Comment:
would adding `*.py` work better?

Reply:
Yeah, that would be a lot less lines of code!